src/main/java/com/algorand/algosdk/v2/client/indexer/SearchForTransactions.java

```diff
@@ -0,0 +1,288 @@
+		return this;
+	}
+
+	/**
+	 * Asset ID 
+	 */
+	public SearchForTransactions setAssetId(long assetId) {
+		this.assetId = assetId;
+		this.assetIdIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Include results before the given time. Must be an RFC 3339 formatted string. 
+	 */
+	public SearchForTransactions setBeforeTime(String beforeTime) {
+		this.beforeTime = beforeTime;
+		this.beforeTimeIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Results should have an amount greater than this value. MicroAlgos are the 
+	 * default currency unless an asset-id is provided, in which case the asset will be 
+	 * used. 
+	 */
+	public SearchForTransactions setCurrencyGreaterThan(long currencyGreaterThan) {
+		this.currencyGreaterThan = currencyGreaterThan;
+		this.currencyGreaterThanIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Results should have an amount less than this value. MicroAlgos are the default 
+	 * currency unless an asset-id is provided, in which case the asset will be used. 
+	 */
+	public SearchForTransactions setCurrencyLessThan(long currencyLessThan) {
+		this.currencyLessThan = currencyLessThan;
+		this.currencyLessThanIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Combine with address and address-role parameters to define what type of address 
+	 * to search for. The close to fields are normally treated as a receiver, if you 
+	 * would like to exclude them set this parameter to true. 
+	 */
+	public SearchForTransactions setExcludeCloseTo(boolean excludeCloseTo) {
+		this.excludeCloseTo = excludeCloseTo;
+		this.excludeCloseToIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Maximum number of results to return. 
+	 */
+	public SearchForTransactions setLimit(long limit) {
+		this.limit = limit;
+		this.limitIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Include results at or before the specified max-round. 
+	 */
+	public SearchForTransactions setMaxRound(long maxRound) {
+		this.maxRound = maxRound;
+		this.maxRoundIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Include results at or after the specified min-round. 
+	 */
+	public SearchForTransactions setMinRound(long minRound) {
+		this.minRound = minRound;
+		this.minRoundIsSet = true;
+		return this;
+	}
+
+	/**
+	 * The next page of results. Use the next token provided by the previous results. 
+	 */
+	public SearchForTransactions setNext(String next) {
+		this.next = next;
+		this.nextIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Specifies a prefix which must be contained in the note field. 
+	 */
+	public SearchForTransactions setNotePrefix(String notePrefix) {
+		this.notePrefix = notePrefix;
+		this.notePrefixIsSet = true;
+		return this;
+	}
+
+	/**
+	 * Include results for the specified round. 
+	 */
+	public SearchForTransactions setRound(long round) {
+		this.round = round;
+		this.roundIsSet = true;
+		return this;
+	}
+
+	/**
+	 * SigType filters just results using the specified type of signature: * sig - 
+	 * Standard * msig - MultiSig * lsig - LogicSig 
```

Comment:
Looks like the newlines are stripped in the generated code.

Reply:
These are coming from the spec file. Should I expect formatting i.e. new line "\n" in the spec file description?

Reply:
Yes, the spec file has this:
```
        "type": {
          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig",
          "type": "string",
          "enum": [
            "sig",
            "msig",
            "lsig"
          ]
        }
```